From: Simon McVittie <smcv@debian.org>
Date: Wed, 11 Aug 2021 14:02:21 +0100
Subject: build: libdecor-0.pc depends on wayland-client.pc

Otherwise we cannot #include <wayland-client.h>.

Signed-off-by: Simon McVittie <smcv@debian.org>
Bug: https://gitlab.gnome.org/jadahl/libdecor/-/issues/26
Forwarded: https://gitlab.gnome.org/jadahl/libdecor/-/merge_requests/78
Applied-upstream: 0.1.1, commit:940f8ea9e22bd159063d398f0eb627b0130f5d3c
---
 src/meson.build | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/meson.build b/src/meson.build
index e0d4f53..52b8f54 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -112,6 +112,9 @@ pkg.generate(
   name: libdecor_full_name,
   description: 'library for Wayland client-side window decors',
   libraries: libdecor,
+  requires_private: [
+    'wayland-client',
+  ],
   subdirs: libdecor_full_name,
   version: meson.project_version()
 )
